Desktop/v2.2.0/Internal_Share_Enhancement_Template.md

```diff
@@ -0,0 +1,18 @@
+### Internal Share Enhancement
+
+
+Server version: 9.1.0 Alpha <br> 
+Client Version: 2.2.0
+
+<br>
+ 
+Test Case ID | Test case     | Steps   | Expected behavior | Result | Related Comments
```

Comment:
Test Case ID column is annoying for maintaining tests plans in this format, not needed.